src/modules/control_allocator/ActuatorEffectiveness/ActuatorEffectivenessFixedWing.cpp

```diff
@@ -61,6 +62,25 @@ ActuatorEffectivenessFixedWing::getEffectivenessMatrix(Configuration &configurat
 	return (rotors_added_successfully && surfaces_added_successfully);
 }
 
+void ActuatorEffectivenessFixedWing::updateSetpoint(const matrix::Vector<float, NUM_AXES> &control_sp,
+		int matrix_index, ActuatorVector &actuator_sp, const matrix::Vector<float, NUM_ACTUATORS> &actuator_min,
+		const matrix::Vector<float, NUM_ACTUATORS> &actuator_max)
+{
+	// Stop front facing motors when they are commanded 0 thrust
```

Comment:
What does this change to the current implementation? It now goes to the configured MIN_PWM when thrust is zero, default 1000: 
![image](https://user-images.githubusercontent.com/26798987/234785994-a53f07ce-654a-4fca-9dd8-e22fc8f568c2.png)

Reply:
It defaults to disarmed 1000, min 1100, max 1900 for **any** motor. And instead of setting the PWM_MIN back to 1000 for specific use cases the plan is to explicitly stop motors when desired with the existing infrastructure so a default fixed-wing pusher has PWM range 1100 - 1900 but can explicitly turn off the motor with low enough commanded thrust. So it's very clear that if you command a thrust the motor turns and if you command it to stop it stops and doesn't depend on the ESC e.g. PWM config, DroneCAN, DSHOT, ...